airgun/entities/trend.py

```diff
@@ -0,0 +1,103 @@
+from navmazing import NavigateToSibling
+from widgetastic.exceptions import RowNotFound
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.trend import (
+    TrendCreateView,
+    TrendEditView,
+    TrendsView,
+)
+
+
+class TrendEntity(BaseEntity):
+
+    def create(self, values):
+        """Create new trend in the system
+
+        Please note that we have static amount of Trend types and trendables, so it is not
+        possible to generate values for them. In case you want to interact with them through
+        automation, it is must have condition to provide unique elements from corresponding
+        lists. In most cases it will not be possible to re-run tests due condition of
+        uniqueness described above.
```

Comment:
This wording seems a little too strong. If test creates new trend and properly deletes it, it can be re-run multiple times in sequence, can't it? Of course if test fails after creating trend, manual intervention is needed to re-run.

Reply:
yes, that means if you delete -> you get your unique element back